### PR TITLE
OL7 pci-dss auditd audit rules

### PR DESCRIPTION
**Pull request by iokomin**
#### Description:
Another set of audit rules adopted for ol7 pci-dss profile including shared content unification changes.

- Inspired by issue #3139 discussion decided to merge rhel6_perform_audit_adjtimex_settimeofday_stime_remediation and rhel7_fedora_perform_audit_adjtimex_settimeofday_stime_remediation into one perform_audit_adjtimex_settimeofday_stime_remediation applicable for rhel6,rhel7, fedora and ol7 products
- Set of rhel6 and rhel7 remediation scripts merged into shared adding fedora and ol7 to supported platforms (as augenrules available in all mentioned distros)
- template_OVAL_audit_rules_usergroup_modification updated to be applicable for multi_platform_rhel and multi_platform_ol

- Enabled rules
  - audit_rules_time_adjtimex
  - audit_rules_time_settimeofday
  - audit_rules_time_stime
  - audit_rules_time_clock_settime
  - audit_rules_time_watch_localtime
  - audit_rules_usergroup_modification_group
  - audit_rules_usergroup_modification_gshadow
  - audit_rules_usergroup_modification_opasswd
  - audit_rules_usergroup_modification_passwd
  - audit_rules_usergroup_modification_shadow
  - audit_rules_networkconfig_modification

Testing successful on Oracle Linux 7

### CODE REVIEW

File: shared/bash_remediation_functions/perform_audit_adjtimex_settimeofday_stime_remediation.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@ source fix_audit_syscall_rule.sh
 
 # Perform the remediation for the 'adjtimex', 'settimeofday', and 'stime' audit
 # system calls on Red Hat Enterprise Linux 7 or Fedora OSes
```
Comment (cipherboy) [inline shared/bash_remediation_functions/perform_audit_adjtimex_settimeofday_stime_remediation.sh]: Updating this comment might be nice.

Comment (iokomin) [reply shared/bash_remediation_functions/perform_audit_adjtimex_settimeofday_stime_remediation.sh]: @cipherboy, sounds reasonable. Please find updated description.

**Comment by cipherboy** [comment]
@openscap-ci test this please.

**Comment by cipherboy** [comment]
@iokomin Thanks for these changes!

```
< Two Plus Objects Specifics - Left Changed:
< 	rule_id:audit_rules_networkconfig_modification has two or more bash remediations: rhel6.sh,rhel7.sh
< 	rule_id:audit_rules_time_adjtimex has two or more bash remediations: rhel6.sh,rhel7.sh
< 	rule_id:audit_rules_time_settimeofday has two or more bash remediations: rhel6.sh,rhel7.sh
< 	rule_id:audit_rules_time_stime has two or more bash remediations: rhel6.sh,rhel7.sh
< 	rule_id:audit_rules_time_watch_localtime has two or more bash remediations: rhel6.sh,rhel7.sh
```
^ thanks! :) 

And these changes help too, I appreciate that:

```
< Prodtypes Objects Specifics - Left Changed:
< 	rule_id:audit_rules_networkconfig_modification has a different prodtypes between YAML and OVALs: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_networkconfig_modification has a different prodtypes between YAML and bash remediations: debian8,example,fedora,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_adjtimex has a different prodtypes between YAML and OVALs: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_adjtimex has a different prodtypes between YAML and bash remediations: debian8,example,fedora,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_clock_settime has a different prodtypes between YAML and OVALs: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_clock_settime has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_settimeofday has a different prodtypes between YAML and OVALs: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_settimeofday has a different prodtypes between YAML and bash remediations: debian8,example,fedora,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_stime has a different prodtypes between YAML and OVALs: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_stime has a different prodtypes between YAML and bash remediations: debian8,example,fedora,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_watch_localtime has a different prodtypes between YAML and OVALs: debian8,example,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
< 	rule_id:audit_rules_time_watch_localtime has a different prodtypes between YAML and bash remediations: debian8,example,fedora,ocp3,ol7,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux


> Prodtypes Objects Specifics - Right Changed:
> 	rule_id:audit_rules_networkconfig_modification has a different prodtypes between YAML and OVALs: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_networkconfig_modification has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_adjtimex has a different prodtypes between YAML and OVALs: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_adjtimex has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_clock_settime has a different prodtypes between YAML and OVALs: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_clock_settime has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_settimeofday has a different prodtypes between YAML and OVALs: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_settimeofday has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_stime has a different prodtypes between YAML and OVALs: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_stime has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_watch_localtime has a different prodtypes between YAML and OVALs: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
> 	rule_id:audit_rules_time_watch_localtime has a different prodtypes between YAML and bash remediations: debian8,example,ocp3,opensuse,rhel-osp7,sle11,sle12,ubuntu1404,ubuntu1604,wrlinux
```


Running testcases locally and then I'll merge when the Jenkins build finishes.

**Pull request merged**